moddio/moddio2
# fix: should move origin coordinates on client-side

## Summary
- for merging many editMapTile actions into one, we need a same origin coordinates, but I forgot to move origin coordinates on client-side and I fix it now.
## Why
- editMapTile will edit tiles in wrong position now
## How to test
[wrongTilePositionTest.txt](https://github.com/moddio/moddio2/files/12848541/wrongTilePositionTest.txt)
- let me explain, u need to open two tabs and just move the homies, 
- let one of the homie go to the upper left corner and pass the another one
if u see the tiles in their position changes, then everything works fine
### and why need to test that
the origin coordinates won't move on client-side before, if u cross the origin coordinates ( in these case, it's the upper left homie's  position ), it can't put tiles anymore (because in that case, the editing position will be negative number which is invalid)
